### PR TITLE
Transparecimeter

### PR DESCRIPTION
**Pull request by luevanio**
I spend some time developing an api that returns the Transperancy index by country. My goal was to create a grathic(named "**transparencymeter**") to mesure the grade of the transperancy index in each country on PALOPS... and some others "gambiarras"

### CODE REVIEW

File: api/api.py [hunk 1]
```diff
@@ -206,6 +207,10 @@ class OutputExpensesRevenueYearFilterSerializer(serializers.Serializer):
     expense_group = serializers.CharField()
 
 
+def requests(country):
```
Comment (pabdelhay) [inline api/api.py:210]: Remove unused functions. Also 'requests' is a reserved word. Never use it as a function

File: apps/budget/models/forms.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    ('MZ', 'Mozambique'),
+    ('ST', 'São Tomé e Principe'),
+    ('GW', 'Guiné-Bissau'),
+    ('TL', 'Timor-Leste')
+)
+
+
+class TransForm(forms.ModelForm):
+    class Meta:
+        model = transparencymeter
```
Comment (pabdelhay) [inline apps/budget/models/forms.py:17]: model is always with CamelCase.

File: apps/budget/models/transparencymeter.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    ('AO', 'Angola'),
+    ('CV', 'Cabo-Verde'),
+    ('MZ', 'Mozambique'),
+    ('ST', 'São Tomé e Principe'),
+    ('GW', 'Guiné-Bissau'),
+    ('TL', 'Timor-Leste')
+)
+
+
+class transparencymeter(models.Model):
```
Comment (pabdelhay) [inline apps/budget/models/transparencymeter.py:13]: Classes are always in CamelCase

File: frontend/templates/frontend/index.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-{% extends 'frontend/base.html' %}
+index.html{% extends 'frontend/base.html' %}
```
Comment (pabdelhay) [inline frontend/templates/frontend/index.html:1]: index.html doesn't make sense. First code of .html shoud be {% extends ,,,

File: frontend/views.py [hunk 5]
```diff
@@ -76,3 +77,19 @@ def get(self, request, *args, **kwargs):
             "country": Country.objects.get(slug=self.kwargs.get("slug"))
         }
         return render(request, 'frontend/expenses-and-revenues.html', context=cn)
+
+
+class NewPage(View):
```
Comment (pabdelhay) [inline frontend/views.py:82]: Name your views meaningfully. 'NewPage' does not tells what this view represents.

**Review by pabdelhay — CHANGES_REQUESTED** [review]

File: api/api.py (hunk 1 above)
Comment (luevanio) [reply api/api.py:210]: done

**Pull request closed without merging**